traits.build/README.md

```diff
@@ -0,0 +1,21 @@
+This W3ID provides a persistent URI namespace for the traits.build data model.
+
+# traits.build
+
+traits.build is the data model for trait databases built using the {traits.build} R-package (https://github.com/traitecoevo/traits.build). The data model both documents the meaning of each of the columns in the relational tables in a database and the semantic relationship between individual tables, columns, rows, and individual data cells. It adheres to the OBOE ontology (https://bioportal.bioontology.org/ontologies/OBOE) and draws upon previously published terms whenever possible, especially DarwinCore (https://dwc.tdwg.org/terms/) and the Ecological Trait-data Standard  (https://ecologicaltraitdata.github.io/ETS/). traits.build includes machine-readable formats and endpoints, allowing the traits to be readily re-used by other databases.
```

Comment:
```suggestion
traits.build is the data model for trait databases built using the {`traits.build`} R-package (https://github.com/traitecoevo/traits.build). The data model documents both the meaning of each of the columns in the relational tables in a database and the semantic relationship between individual tables, columns, rows, and individual data cells. It adheres to the OBOE ontology (https://bioportal.bioontology.org/ontologies/OBOE) and draws upon previously published terms whenever possible, especially DarwinCore (https://dwc.tdwg.org/terms/) and the Ecological Trait-data Standard (https://ecologicaltraitdata.github.io/ETS/). traits.build includes machine-readable formats and endpoints, allowing the traits to be readily re-used by other databases.
```

Reply:
Thanks @TallTed, really appreciate these suggestions. I'll send a new PR to update the text later this week.